enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -6425,7 +6846,7 @@ class AdjointGenerator
             freeKnownAllocation(Builder2, lookup(nop, Builder2), *called,
                                 dbgLoc, gutils->TLI);
           }
-        } else if (Mode != DerivativeMode::ReverseModePrimal) {
+        } else {
```

Comment:
This should be != primal and != forward

Reply:
done 5258db043a9b229393ca3d2e2f79232a2bc32b62